core/diirt/diirt-features/pom.xml

```diff
@@ -30,7 +30,6 @@
         <!-- Plugin required to properly publish source feature -->
         <groupId>org.eclipse.tycho</groupId>
         <artifactId>tycho-p2-plugin</artifactId>
-        <version>${tycho.version}</version>
```

Comment:
I think you need the explicit tycho.version (it exists in all other xx-features/pom.xml).
Removing it may result in incompatible tycho-extras and tycho versions being used in the build.